hub/executors/indexers/vector/annoy/annoy.yml

```diff
@@ -0,0 +1,8 @@
+!AnnoyIndexer
+metas:
+  workspace: $TMP_WORKSPACE
+  name: indexer_annoy
+  py_modules: annoy.py
```

Comment:
`AnnoyIndexer` is a built-in class, and therefore you can use it out-of-box. `py_modules` is useful when you want to implement your own executors.
```suggestion
```